tasks/build.py

```diff
@@ -64,9 +64,16 @@ def setup_node(ctx):
             npm_command = 'npm'
 
             if NVMRC_PATH.is_file():
+                # nvm will output the node and npm versions used if nvmrc exists
                 LOGGER.info('Using node version specified in .nvmrc')
                 ctx.run('nvm install')
                 npm_command = f'nvm use && {npm_command}'
+            else:
```

Comment:
@jmhooper: added this section also since I noticed that the node and npm versions are never logged when the default versions are used